app/components/why-input.jsx

```diff
@@ -0,0 +1,40 @@
+// https://github.com/EnCiv/civil-pursuit/issues/68
+
+'use strict'
+import React from 'react';
+import Point from './point';
+import PointInput from './point-input';
+import { createUseStyles } from 'react-jss';
+
+
+function WhyInput(props) {
+    const { point = { subject: "", description: "", vState: "secondary" }, defaultValue = { subject: "", description: "" }, onDone = () => { }, className, ...otherProps } = props;
+    const classes = useStyles();
+
+    const handleOnDone = ({ valid, value }) => {
+        onDone({ valid, value })
+    }
+
+    return (
+        <div className={classes.container} {...otherProps}>
+            <Point {...point} className={className} {...otherProps} />
```

Comment:
otherProps should only be part of the outer div - not passed deeper lie to the Point.  Also, the className in props should be combined with classes.container in the outer div - and not passed to the Point.